app/billing/rest.py

```diff
@@ -60,41 +55,6 @@ def get_yearly_billing_usage_summary_from_ft_billing(service_id):
     return jsonify(data)
 
 
-@billing_blueprint.route('/monthly-usage')
```

Comment:
do we want to, as part of this story, migrate the new routes to have these old endpoint routes? i'm not a huge fan of leaving `/ft-...` as part of the route, it's just a technical implementation detail.

if so, then this PR could make those endpoints accessible on both the old and the new routes (with a future PR removing the ft-monthly-usage and ft-yearly-usage-summary routes after admin is changed)
if not, then this PR is fine.

Reply:
I could do it no problem 👍